cmd/neofs-adm/internal/modules/morph/initialize_transfer.go

```diff
@@ -0,0 +1,150 @@
+package morph
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/client"
+	scContext "github.com/nspcc-dev/neo-go/pkg/smartcontract/context"
+)
+
+const (
+	// initialAlphabetNEOAmount represents total amount of NEO distributed between alphabet nodes.
+	initialAlphabetNEOAmount = native.NEOTotalSupply * 9 / 10
+	gasInitialTotalSupply    = 30000000 * native.GASFactor
+	// initialAlphabetNEOAmount represents total amount of GAS distributed between alphabet nodes.
+	initialAlphabetGASAmount = 30000000 * native.GASFactor / 2
+	// initialCommitteeGASAmount represents amount of GAS transferred to committee.
+	initialCommitteeGASAmount = (gasInitialTotalSupply - initialAlphabetGASAmount) / 2
+)
+
+func (c *initializeContext) transferFunds() error {
+	ok, err := c.transferFundsFinished()
+	if ok || err != nil {
+		if err == nil {
+			c.Command.Println("Stage 1: already performed.")
+		}
+		return err
+	}
+
+	neoHash, err := c.Client.GetNativeContractHash(nativenames.Neo)
+	if err != nil {
+		return fmt.Errorf("can't fetch %s hash: %w", nativenames.Neo, err)
+	}
+	gasHash, err := c.Client.GetNativeContractHash(nativenames.Gas)
+	if err != nil {
+		return fmt.Errorf("can't fetch %s hash: %w", nativenames.Gas, err)
+	}
+
+	var transfers []client.TransferTarget
+	for _, w := range c.Wallets {
+		acc, err := getWalletAccount(w, singleAccountName)
+		if err != nil {
+			return err
+		}
+
+		to := acc.Contract.ScriptHash()
+		transfers = append(transfers,
+			client.TransferTarget{
+				Token:   neoHash,
+				Address: to,
+				Amount:  initialAlphabetNEOAmount / int64(len(c.Wallets)),
+			},
+			client.TransferTarget{
+				Token:   gasHash,
+				Address: to,
+				Amount:  initialAlphabetGASAmount / int64(len(c.Wallets)),
```

Comment:
Probably we need to decrease `initialAlphabetGASAmount`.
We suppose that admin tool will set up notary enabled environment. Therefore most of the initial GAS must go to proxy contract address later.